### ISSUE TITLE
spinner example clarification

### ISSUE THREAD

**Issue by Xoristzatziki** [sebp/PyGObject-Tutorial#118]
The spinner example shows only how to use it in case a user intentionally starts and stops the spinner. Does not show how to use it in real coding. In other words does not show how to indicate something (ex. a calculation) is in progress. Using start() and stop() in the same function (def) will show nothing. i.e.:
```
def calculate_something(self):
    self.spinner.start()
    for i in range(10000):
        dosomething()
    self.spinner.stop()
```

does not show spinning at all...

**Comment by sebp** [comment]
I can see that the example is not very useful for practical purposes. You'd have to perform your computation in a separate thread and not block the main loop. You can achieve this relatively easy by passing your computation-heavy function to [idle_add](https://lazka.github.io/pgi-docs/index.html#GLib-2.0/functions.html#GLib.idle_add). This is currently not covered in the tutorial.

**Comment by Xoristzatziki** [comment]
My comment was not about  how I, personally, will use the spinner, but about adding, somehow, a clarification (such as your commented above) in the documentation found in http://python-gtk-3-tutorial.readthedocs.io/en/latest/spinner.html, which led me here. There is an "edit in github" link there but I do not know how to edit it...

**Comment by sebp** [comment]
Fixed by #128
